README.md

```diff
@@ -1 +1,42 @@
+* The majority of participating reviewers rejects the proposal → The proposal is rejected. The participating reviewers and the author will consider whether any part part of the proposal can be improved and/or incorporated into a new proposal
+Community contributors are encouraged to present proposals and to share their opinions on any Request For Enhancement (RFE). However, any approval throughout the workflow must be granted by the Narayana team and its optional members.
+
+\* The time window can be adjusted according to individual and team circumstances. However, the Narayana team is required to provide a different time window within 3 days from the date a new proposal is created.
+
+## 1. Proposal and technical document
+
+To ensure a clear connection between proposals and their corresponding tasks, a dedicated folder for each proposal will be created within this repository. The folder name will be the unique JIRA issue associated with the RFE. This JIRA issue should be opened before creating the proposal.
+
+In order to facilitate open discussions and transparent collaboration, a pull request for the technical document must be raised for each proposal. The discussion will thus evolve through reviews and feedback. When the proposal is given the go-ahead by the reviewers, the pull request will be merged.
+
+Once the technical document has been approved and merged, it cannot be modified. If any improvement or modification is needed, the new technical document will have to be reviewed and approved by all reviewers.
+
+To facilitate the navigation between the pull request and the related JIRA, a hyperlink to the JIRA should be added to the description of the pull request and, vice versa, a link to the pull request should be attached to the JIRA.
+
+## 2. Implementation Document (optional)
+
+An optional implementation-specific document can be submitted at this stage. This document should contain a clear roadmap to ensure a smooth transition from design to implementation. Although this step is optional, it might become necessary when there are disagreements among reviewers. Any time there is a disagreement on the implementation of the technical document, a pull request for an implementation-specific document must be raised and a discussion should be initiated. Once the Narayana team agrees on a specific implementation, the pull request should be merged and the RFE must be developed based on that document.
+
+Once the implementation-specific document has been approved and merged, it cannot be modified. If any improvement or modification is needed, the new implementation-specific document will have to be reviewed and approved by all reviewers.
```

Comment:
"Once the implementation-specific document has been approved and merged, it cannot be modified without a pull request".

Reply:
Thanks Michael. I prefer to stick with the long version, if you don't mind. I want to make sure that it is clear that a new approval is needed in case any modification is required.

Reply:
My point was that you state that it cannot be modified once its merged but you then go on to say that if a modification is needed then new PR is needed, ie the second statement contradicts the first.

The general issue I have with this proposal is that it is far to prescriptive and that leaves you open to too many contradictions in the text and it raises the bar too high for contributions.

Reply:
The second statement doesn't contradict the first statement. It adds on top of it.

> The general issue I have with this proposal is that it is far to prescriptive and that leaves you open to too many contradictions in the text and it raises the bar too high for contributions.

I'll reply to this concern in your general comment.

Reply:
I have resolved this discussion. Feel free to unresolve it if you need to continue the discussion.